src/librustc_lexer/src/lib.rs

```diff
@@ -102,10 +183,9 @@ pub fn tokenize(mut input: &str) -> impl Iterator<Item = Token> + '_ {
     })
 }
 
-// See [UAX #31](http://unicode.org/reports/tr31) for definitions of these
-// classes.
-
 /// True if `c` is considered a whitespace according to Rust language definition.
+/// See [Rust language reference](https://doc.rust-lang.org/reference/whitespace.html)
```

Comment:
"Rust language reference" and "Rust reference" are both used here in the comments, perhaps normalize to one variant.